Add White belt activity: Think about where the journey is going (#27)
AppSecure-nrw/security-belts: white/think-about-where-the-journey-is-going.md

# Think about where the journey is going

> Think about which security belt might be sufficient for the business criticality of your software.

Reviewer: ```suggestion
Estimate which security belt might be sufficient for the business criticality of your software.
```

> Take a look at:
> 
> - which kind of data is processed by your software.
> - how important your software is for your core business.
> - your compliance and regulatory requirements.

Reviewer: ```suggestion
- To estimate, take a look at:
  - which kind of data is processed by your software.
  - how important your software is for your core business.
  - your compliance and regulatory requirements.
- Agree with your product owner which Security Belt your software deserves.
```

# Think about where the journey is going

> Achieving the highest Security Belt might not make economic sense and is therefore not necessary.

Author: ```suggestion
Achieving the highest Security Belt might not make economic sense and may therefore not be necessary.
```

> Depending on the criticality of your own software, achieving the highest Security Belt might not make economic sense and is therefore not necessary.

Author: ```suggestion
Achieving the highest Security Belt might not make economic sense and is therefore not necessary.
```

> Depending on the criticality of your own software, achieving the highest Security Belt might not make economic sense and is therefore not necessary.

Author: ```suggestion
Depending on the criticality of your own software, achieving the highest Security Belt might not make economic sense and may therefore not be necessary.
```

> Depending on the criticality of your own software, achieving the highest Security Belt might not make economic sense and is therefore not necessary.

Author: ```suggestion
Depending on the criticality of your own software, achieving the highest Security Belt might not make economic sense and may therefore not be necessary.
```

> - Estimate which Security Belt might be sufficient for the business criticality of your software.

Author: ```suggestion
- Estimate which Security Belt might be sufficient for the business criticality of your software. Your estimate might be just a first guess but not the last.
```

> - Estimate which Security Belt might be sufficient for the business criticality of your software. Your estimate might be just a first guess but not the last.

Author: ```suggestion
- Estimate which Security Belt might be sufficient for the business criticality of your software. Your estimate might be just a first guess.
```

- Commit on your estimation with your product owner.

> - Challenge your estimation after each achieved Security Belt. Since you should have a better understanding of security, your estimation should be more profound.

Author: ```suggestion
- Challenge your estimation after each achieved Security Belt. Since you should have a better understanding of security, your estimation should become more profound.
```
